newsfragments/42285.significant.rst

```diff
@@ -0,0 +1 @@
+The SLA feature is removed in Airflow 3.0, to be replaced wAirflow 3.0 removes the SLA feature, which will be replaced with a new implementation of the feature in 3.1 2ith a new implementation in 3.1
```

Comment:
Something went wrong with the text? Pasted 3 times? Too thick fingers on the keyboard?

Reply:
Well that is just embarrassing.  Fixed and rephrased.